pkg/controller/installation/prometheusRules.go

```diff
@@ -64,12 +64,12 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 					{
 						Alert: fmt.Sprintf("%sOperatorInstallDelayed", strings.ToUpper(installationName)),
 						Annotations: map[string]string{
-							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
+							"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/OperatorInstallDelayed.asciidoc",
```

Comment:
would it be better to follow same approach as for other alerts and create a variable that holds the url?

Reply:
Sure, will change that now